### DISCUSSION TITLE
Upload file dialog way too sensitive

### DISCUSSION THREAD

**Discussion by traverseda** [open-webui/open-webui#809] [General]
With open-webui v1.0.0-alpha.100, and firefox on wayland kde, whenever you copy text from the website or the website tab loses focus the add-files dialog pops up, and won't go away.

Not sure what's causing this, but it's pretty persistently annoying as I end up needing to refresh the page every time I copy something.

**Comment by traverseda** [comment]
This may be a bug in wayland-firefox

**Comment by tjbck** [comment]
I highly suspect that this is caused by wayland-firefox. FYI, There was someone who reported the same issue on discord, and they resolved the issue by removing some extensions I believe.
